samples/bluetooth/bap_broadcast_sink/src/main.c

```diff
@@ -1248,10 +1248,32 @@ static void bap_pa_sync_terminated_cb(struct bt_le_per_adv_sync *sync,
 				      const struct bt_le_per_adv_sync_term_info *info)
 {
 	if (sync == pa_sync) {
-		printk("PA sync %p lost with reason %u\n", sync, info->reason);
+		printk("PA sync %p lost with reason 0x%08x\n", sync, info->reason);
```

Comment:
```suggestion
		printk("PA sync %p lost with reason 0x%02x\n", sync, info->reason);
```

The reason field should only ever be 8-bits :)